deno2/deno.cc

```diff
@@ -251,7 +251,7 @@ v8::StartupData MakeSnapshot(v8::StartupData* prev_natives_blob,
   }
 
   auto snapshot_blob =
-      creator->CreateBlob(v8::SnapshotCreator::FunctionCodeHandling::kKeep);
+      creator->CreateBlob(v8::SnapshotCreator::FunctionCodeHandling::kClear);
```

Comment:
I would suggest to add a comment here, noting that (maybe) kKeep would be better if it didn't make v8 crash.

Looking at mksnapshot, it uses:
* `kClear` in `CreateSnapshotDataBlob()`
* `kKeep` in `WarmUpSnapshotDataBlob()`

Reply:
Comment added. I believe kKeep is meant for usage with WarmUps. Let's see what V8 says about it.